rtcpeerconnection.js

```diff
@@ -636,11 +640,15 @@ module.exports = function(window, edgeVersion) {
           delete p.rtx;
         });
       }
-      params.encodings = transceiver.recvEncodingParameters;
```

Comment:
see above

Reply:
ignore the "see above". You're right, passing undefined is better than passing an empty array:
http://draft.ortc.org/#dom-rtcrtpparameters-encodings
"When unset in a call to send()". Arguably it could handle an empty array but.. ;-)